dnf/base.py

```diff
@@ -1289,16 +1293,20 @@ def install(self, pkg_spec, reponame=None):
                 self._goal.install(a)
             return len(available)
         elif self.conf.multilib_policy == "best":
-            sltr = subj.get_best_selector(self.sack)
-            if not sltr.matches():
+            sltrs = subj.get_best_selectors(self.sack)
+            match = reduce(lambda x,y: y.matches() or x, sltrs, [])
```

Comment:
And it would follow [our conventions](https://github.com/rpm-software-management/dnf/wiki/Hacking#coding-style) that are: "Adhere to [PEP 008](https://www.python.org/dev/peps/pep-0008/#other-recommendations) [...] as much as reasonable."